app/components/DocsLayout.tsx

```diff
@@ -321,6 +322,10 @@ export function DocsLayout({
   const versionConfig = useVersionConfig({ versions })
   const menuConfig = useMenuConfig({ config, frameworks, repo })
 
+  const { mode: themeMode } = useThemeStore()
+
+  const oramaThemeMode = themeMode === 'auto' ? 'dark' : themeMode
+
```

Comment:
You could detect auto here with a media match as done in other areas

Reply:
Hi @tannerlinsley nice catch.
Actually Orama components does it internally when "system" is set to colorScheme.
I've updated the code so "auto" translates to "system". Is it good enough?

Reply:
@tannerlinsley, hey! Just checking if this is enough for you request. Thanks!